chart/values.yaml

```diff
@@ -2853,9 +2853,13 @@ cleanup:
   successfulJobsHistoryLimit: ~
 
 # Configuration for postgresql subchart
-# Not recommended for production
+# Uses bitnamilegacy images to avoid Bitnami licensing restrictions
+# Not recommended for production - use external database instead
 postgresql:
   enabled: true
+  image:
+    repository: bitnamilegacy/postgresql
+    tag: "13.2.0"
```

Comment:
```suggestion
    repository: postgres
    tag: "13"
```
Perhaps?

Reply:
I can confirm that it works, so let me make that change, let me know otherwise.

Reply:
I frankly never knew why we used the bitnami image in the first place